iothub/service/src/ExportImportDevice.cs

```diff
@@ -11,27 +11,51 @@
 namespace Microsoft.Azure.Devices
 {
     /// <summary>
-    ///  contains device properties specified during export/import operation
+    /// Contains device properties specified during export/import operation
     /// </summary>
     public sealed class ExportImportDevice
     {
         private string _eTag;
         private string _twinETag;
 
         /// <summary>
-        /// making default ctor public
+        /// Property container
+        /// </summary>
+        [System.Diagnostics.CodeAnalysis.SuppressMessage("Design", "CA1034:Nested types should not be visible", Justification = "Publicly property. No behavior changes allowed.")]
+        public sealed class PropertyContainer
+        {
+            /// <summary>
+            /// Desired properties
+            /// </summary>
+            [JsonProperty(PropertyName = "desired", NullValueHandling = NullValueHandling.Ignore)]
+            public TwinCollection DesiredProperties { get; set; }
+
+            /// <summary>
+            /// Reported properties
+            /// </summary>
+            [JsonProperty(PropertyName = "reported", NullValueHandling = NullValueHandling.Ignore)]
+            public TwinCollection ReportedProperties { get; set; }
+        }
+
+        /// <summary>
+        /// Create an ExportImportDevice
         /// </summary>
         public ExportImportDevice()
         {
         }
 
         /// <summary>
-        /// ctor which takes a Device object along with import mode
+        /// Create an ExportImportDevice
```

Comment:
use <see cref="ExportImportDevice" />

Reply:
That will be a pointer to the same file .. what's the benefit ?

Reply:
True. It's just easier to make sure the comment is not outdated if we rename the class. I also see you used `<see cref="ExportImportDevice" />,` in the ExportImportDevice.cs class, so I thought maybe keep same pattern.